### PR TITLE
add Golang stepik course and browser IDE

### PR DESCRIPTION
**Pull request by samurai-of-honor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # Languages
 💻 An open-access repository to store resources for learning various programming languages and technologies.
+
+## JavaScript
```
Comment (SergeyOcheretenko) [inline README.md:4]: Change all title to ### mode, it's more beautiful and easy for eyes

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+
+## Python
```
Comment (SergeyOcheretenko) [inline README.md:6]: Change all title to ### mode, it's more beautiful and easy for eyes

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+
+## C#.
```
Comment (SergeyOcheretenko) [inline README.md:8]: Change all title to ### mode, it's more beautiful and easy for eyes

File: README.md [hunk 4]
```diff
@@ -1,2 +1,14 @@
...
+
+## Java
```
Comment (SergeyOcheretenko) [inline README.md:10]: Change all title to ### mode, it's more beautiful and easy for eyes

**Review by SergeyOcheretenko — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (SergeyOcheretenko) [inline README.md:8]: Delete dot near C#

File: README.md [hunk 5]
```diff
@@ -1,2 +1,14 @@
...
+
+## Go
++ [Stepik](./Go/README.md#stepik)
```
Comment (SergeyOcheretenko) [inline README.md:13]: Change "Stepik" to "Stepik Courses"

**Review by SergeyOcheretenko — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (samurai-of-honor) [reply README.md:8]: If i delete dot - # will not be displayed. What should I do?

Comment (SergeyOcheretenko) [reply README.md:8]: Add backslash before #:
C\#

Comment (samurai-of-honor) [reply README.md:8]: My mistake. For some reason it didn't work in my editor

**Review by SergeyOcheretenko — APPROVED** [review]

**Pull request merged**